pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/types"
 )
 
+const (
+	// PbrNatOutGatewayPrio : IP rule priority for packets destined to gateway(bridge ip) coming from apps.
+	PbrNatOutGatewayPrio = 9999
+	// PbrNatOutPrio : IP rule priority for packets destined to internet coming from apps
+	PbrNatOutPrio = 10000
+	// PbrNatInPrio : IP rule priority for external packets coming in towards apps
+	PbrNatInPrio = 11000
```

Comment:
These apply to Local and Cloud network instances I think. So the "Nat" part of the name might be misleading. Calling them "App" instead of "Nat"? Or "NI"?

Reply:
Ah, they only apply to the Local network instance based on the code?

Reply:
Yes they only apply to NAT instances.